lib/optimize/ConcatenatedModule.js

```diff
@@ -647,7 +647,7 @@ class ConcatenatedModule extends Module {
 		modules,
 		runtime,
 		associatedObjectForCache,
-		hashFunction = "md4"
+		hashFunction = "xxhash64"
```

Comment:
```suggestion
		hashFunction
```

Maybe make it non-optional, similar to the other changes